test/util-test.js

```diff
@@ -219,6 +219,21 @@ describe('util.between()', function() {
 });
 
 
+describe('util.objectAssign()', function() {
+  if('Merges to objects into one', function() {
+    var obj1 = {key1:'value1', key2:true}
+    var obj2 = {key2:false, key3:[1,2,{3:3}], key4:'value4'}
+    var mergedObject = util.objectAssign(obj1,obj2)
+    assert.deepEqual(mergedObject, {
+      key1:'value1',
+      key2:false,
+      key3:[1,2,{3:3}],
+      key4:'value4'
```

Comment:
Spacing around these object declarations doesn't match the style for the rest of the project. The project uses spaces after `{` and before `}` for objects on the same line. And spaces after `:` and `,`, except at the end of the line.